test/sender.spec.ts

```diff
@@ -42,6 +42,18 @@ describe("Sender", () => {
     assert.isFalse(sender.isOpen(), "Sender should not be open.");
   });
 
+  it("Delivery returned from `AwaitableSender.send()` is not undefined", async () => {
+    const sender = await connection.createAwaitableSender({
+      sendTimeoutInSeconds: 1,
+    });
+    const response = await sender.send({ body: "message" });
```

Comment:
Should we also assert any of the fields on the response? Maybe delivery id?

Reply:
Done